python/sparkdl/transformers/named_image.py

```diff
@@ -187,6 +198,22 @@ def setScaleHint(self, value):
     def getScaleHint(self):
         return self.getOrDefault(self.scaleHint)
 
+    @classmethod
+    def read(cls):
+        return _DeepImageFeaturizerReader(cls)
+
+    @classmethod
+    def _from_java(cls, java_stage):
```

Comment:
Is https://github.com/apache/spark/blob/41bb1ddc63298c004bb6a6bb6fff9fd4f6e44792/python/pyspark/ml/wrapper.py#L218 the reason we can't use the default implementation? (e.g. if the class loader logic was in a separate method we could just overwrite that?)

Reply:
yes.